src/main/java/io/vertx/proton/ProtonTransportOptions.java

```diff
@@ -21,50 +21,86 @@
 public class ProtonTransportOptions {
 
   private int heartbeat;
+  private int maxFrameSize;
 
   /**
-   * Set the heartbeat as maximum delay between sending frames for the remote peers.
-   * If no frames are received within 2*heartbeat, the connection is closed
+   * Set the heart beat as maximum delay between sending frames for the remote peers.
+   * If no frames are received within 2 * heart beat, the connection is closed
    *
-   * @param heartbeat hearthbeat maximum delay
-   * @return  current ProtonTransportOptions instance
+   * @param heartbeat The maximum delay in milliseconds.
+   * @return current ProtonTransportOptions instance.
    */
   public ProtonTransportOptions setHeartbeat(int heartbeat) {
     this.heartbeat = heartbeat;
     return this;
   }
 
   /**
-   * Return the heartbeat as maximum delay between sending frames for the remote peers.
+   * Returns the heart beat as maximum delay between sending frames for the remote peers.
    *
-   * @return  hearthbeat maximum delay
+   * @return The maximum delay in milliseconds.
    */
   public int getHeartbeat() {
     return this.heartbeat;
   }
 
+  /**
+   * Sets the maximum frame size to announce in the AMQP <em>OPEN</em> frame.
+   * <p>
+   * If this property is not set explicitly, a reasonable default value is used.
+   * <p>
+   * Setting this property to -1 will result in no maximum frame size being announced at all.
```

Comment:
Perhaps say "negative" rather than "to -1" to match what it appears the impl does?